legacy/src/app/partials/nodes-list.html

```diff
@@ -715,6 +715,7 @@ <h4 class="page-header__dropdown-title">Add device</h4>
             <h4 class="page-header__dropdown-title">Add rack controller</h4>
             <pre><code># To add a new rack controller, SSH into the rack controller.
 # Install the maas-rack-controller package.
+# Please confirm maas version is the same as main rack controller first.
```

Comment:
Just saying: "Confirm maas version is..." should be enough, no need for 'Please'